.github/workflows/ci.yml

```diff
@@ -38,6 +38,26 @@ jobs:
         if: ${{ matrix.buildType == 'Release' }}
         run: |
           time make test262
+  linux-32bits:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: jirutka/setup-alpine@v1
```

Comment:
Just curious, why alpine and not e.g. multi-arch ubuntu?

Reply:
Seemed easy and we test an extra Linux variant? :-P